src/main/java/daronek6/cinema/services/TicketService.java

```diff
@@ -0,0 +1,35 @@
+package daronek6.cinema.services;
+
+import daronek6.cinema.entities.Ticket;
+import daronek6.cinema.repositories.TicketRepo;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class TicketService {
+
+    private final TicketRepo repo;
+
+    public Ticket add(Ticket ticket) {
+        return repo.save(ticket);
+    }
+    public double getTotal(Ticket[] tickets) {
+        double total = 0.0;
+        for (Ticket ticket : tickets) {
+            total += ticket.getPrice();
+        }
+        return total;
+    }
+
+    public double getTotal(Long adultTickets, Long studentTickets, Long childrenTickets) {
+        List<Ticket> currentTickets = repo.findAllByPriceEndTime(null);
```

Comment:
To miał być mechanizm elastycznych cen biletów? Np. większe w godzinach wieczornych albo w weekendy?

Reply:
Wyszukuje aktualne ceny biletów. Pomysł dokładniej opisany pod Ticket.java